src/api.ts

```diff
@@ -41,6 +41,7 @@ export async function synchronize(
     'skip-migration': false,
     ...options
   };
+  const { onStarted, ...invokeParams } = params;
```

Comment:
No ts-lint `unused-variable` errors?

```suggestion
  const { onStarted: _, ...invokeParams } = params;
```